udata/api/__init__.py

```diff
@@ -100,6 +100,9 @@ def _apply_secure(self, func, permission=None):
 
         @wraps(func)
         def wrapper(*args, **kwargs):
+            if current_app.config['READ_ONLY_MODE'] and any(ext in str(func) for ext in current_app.config['METHOD_BLOCKLIST']):
```

Comment:
Can we add a super admin condition here? Super admins should always be allowed to do everything.